lib/cpp/src/thrift/server/TNonblockingServer.cpp

```diff
@@ -1524,7 +1532,7 @@ void TNonblockingIOThread::breakLoop(bool error) {
   }
 
   // sets a flag so that the loop exits on the next event
-  event_base_loopbreak(eventBase_);
+  event_base_loopexit(eventBase_, 0);
```

Comment:
I am curious to understand why this was changed.  event_base_loopexit will cause the lop to continue processing until drained.  This function is called breakLoop and I assume that means stop as soon as possible, which is what it was doing.